src/bindings/python/src/pyopenvino/core/infer_request.cpp

```diff
@@ -65,11 +65,9 @@ void regclass_InferRequest(py::module m) {
             self._start_time = Time::now();
             self._request.infer();
             self._end_time = Time::now();
-            Containers::InferResults results;
-            for (auto& out : self._outputs) {
-                results.push_back(self._request.get_tensor(out));
-            }
-            return results;
+
+            py::dict res = Common::outputs_to_dict(self._outputs, self._request);
```

Comment:
you may also just `return Common::outputs_to_dict(self._outputs, self._request);`